packages/protocol/contracts/authorizers/interfaces/IClientWhitelister.sol

```diff
@@ -1,9 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.6;
 
-import "./IRrpAuthorizer.sol";
-
-interface IClientWhitelistRrpAuthorizer is IRrpAuthorizer {
+interface IClientWhitelister {
     enum AdminRank {
```

Comment:
One other thing, isn't it a bad practice to define enums and structs in interfaces?

Reply:
I'm not sure if its a bad practice or not, but It was added a while back and no one really pointed any issues in the pull request: https://github.com/ethereum/solidity/issues/4087 . Care to share the link where its pointed out as a bad practice?

Reply:
https://medium.com/upstate-interactive/solidity-how-to-know-when-to-use-abstract-contracts-vs-interfaces-874cab860c56

Reply:
You are right, I was reading a medium post which was using an older version of solidity as reference. https://medium.com/upstate-interactive/solidity-how-to-know-when-to-use-abstract-contracts-vs-interfaces-874cab860c56